chain-tx-enclave/enclave-t-common/Cargo.toml

```diff
@@ -7,10 +7,10 @@ readme = "../../README.md"
 edition = "2018"
 
 [dependencies]
-sgx_tstd    = { rev = "5b5e512abbbfc3030e8cb274989fb372b685b168", git = "https://github.com/baidu/rust-sgx-sdk.git" }
+sgx_tstd    = { rev = "v1.1.0", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
```

Comment:
It seems `teaclave-sgx-sdk` get renamed into `incubator-teaclave-sgx-sdk` already.

Reply:
@yihuang forked crates use this path: https://github.com/mesalock-linux/rustls/blob/mesalock_sgx/Cargo.toml#L20

Reply:
it may get renamed again later: https://github.com/apache/incubator-teaclave-sgx-sdk/issues/174